docs/src/main/sphinx/installation/cli.rst

```diff
@@ -13,28 +13,182 @@ Requirements
 The CLI requires a Java virtual machine available on the path.
 It can be used with Java version 8 and higher.
 
+The CLI uses the HTTP protocol and the :doc:`Trino client REST API
+</develop/client-protocol>` to communicate with the coordinator on the Trino
+cluster.
+
 Installation
 ------------
 
-Download :maven_download:`cli`, rename it to ``trino``,
-make it executable with ``chmod +x``, then run it:
+Download :maven_download:`cli`, rename it to ``trino``, make it executable with
+``chmod +x``, and run it to show the version of the CLI:
+
+.. code-block:: text
+
+    ./trino --version
+
+Run the CLI with ``--help`` or ``-h`` to see all available options.
+
+Running the CLI
+---------------
+
+The minimal command to start the CLI in interactive mode specifies the URL of
+the coordinator in the Trino cluster:
+
+.. code-block:: text
+
+    ./trino --server http://trino.example.com:8080
+
+If successful, you will get a prompt to run the CLI. Use the ``help`` command to
+see a list of supported commands:
+
+.. code-block:: text
+
+    trino> help
+
+    Supported commands:
+    QUIT
+    EXIT
+    CLEAR
+    EXPLAIN [ ( option [, ...] ) ] <query>
+        options: FORMAT { TEXT | GRAPHVIZ | JSON }
+                TYPE { LOGICAL | DISTRIBUTED | VALIDATE | IO }
+    DESCRIBE <table>
+    SHOW COLUMNS FROM <table>
+    SHOW FUNCTIONS
+    SHOW CATALOGS [LIKE <pattern>]
+    SHOW SCHEMAS [FROM <catalog>] [LIKE <pattern>]
+    SHOW TABLES [FROM <schema>] [LIKE <pattern>]
+    USE [<catalog>.]<schema>
+
+You can now run SQL statements. After processing, the CLI will show results and
+statistics.
+
+Use the ``clear`` command to clear the terminal. To stop and exit the CLI, run
+``exit`` or ``quit``.
+
+.. code-block:: text
+
+  trino> SELECT count(*) FROM tpch.tiny.nation;
+
+  _col0
+  -------
+      25
+  (1 row)
+
+  Query 20220324_213359_00007_w6hbk, FINISHED, 1 node
+  Splits: 13 total, 13 done (100.00%)
+  2.92 [25 rows, 0B] [8 rows/s, 0B/s]
+
+In a typical deployment, authentication is required. The CLI supports multiple
+:ref:`authentication methods <cli-authentication>`. The most common
+authentication method is :ref:`cli-username-password-auth`.
+
+As part of starting the CLI, you can set the default catalog and schema. This
+allows you to query tables directly without specifying catalog and schema.
 
 .. code-block:: text
 
-    ./trino --server localhost:8080 --catalog hive --schema default
+  ./trino --server http://trino.example.com:8080 --catalog tpch --schema tiny
+
+  trino:tiny> SHOW TABLES;
 
-Run the CLI with the ``--help`` option to see the available options.
+    Table
+  ----------
+  customer
+  lineitem
+  nation
+  orders
+  part
+  partsupp
+  region
+  supplier
+  (8 rows)
 
-The CLI uses the HTTP protocol and the
-:doc:`Trino client REST API </develop/client-protocol>` to communicate
-with Trino.
+You can also set the default catalog and schema with the :doc:`/sql/use`
+statement.
```

Comment:
I added this example. Let me know if it isn't necessary.

Reply:
This is really useful. thanks. let's keep this example.